src/amo/components/LandingPage/styles.scss

```diff
@@ -36,28 +29,34 @@
 }
 
 .LandingPage-icon {
-  height: 89px;
-  width: 225px;
+  width: 100%;
```

Comment:
Damn, I thought my stylelint forced alphabetisation of attributes 😆

Reply:
I think it does, I'll fix that up.